### DISCUSSION TITLE
How to disable Lazy Loading

### DISCUSSION THREAD

**Discussion by goplastic** [Laravel-Backpack/community-forum#977] [General]
Hi, i need to know how to prevent lazy loading all the relations in my list view. 
What do i have to add in setupListOperation() to avoid my model EventLog prevent my relation eventable() when visiting list view?
Thank you!

**Comment by goplastic** [comment]
I understand that adding a column that refears to a relation, trigger the eager loading in Read trait. But i need the value of the column without loading the entire model.

**Reply by karandatwani92** [reply]
I'm assuming you are using `eventable` as the column name, and the DB column name is `event_id`.

You can try two things:
1. set column type as `text` or `number`
2. set column name as DB column name, not relationship name.
